### ISSUE TITLE
[BUG] Error when i try to build cppfront

### ISSUE THREAD

**Issue by fenix272** [hsutter/cppfront#1227]
g++ cppfront.cpp -std=c++20 -o cppfront
C:/codeblocks/MinGW/bin/../lib/gcc/x86_64-w64-mingw32/13.1.0/../../../../x86_64-w64-mingw32/bin/as.exe: C:\Users\Dio\AppData\Local\Temp\ccY6j4zz.o: too many sections (41204)
C:\Users\Dio\AppData\Local\Temp\ccsHBHmj.s: Assembler messages:
C:\Users\Dio\AppData\Local\Temp\ccsHBHmj.s: Fatal error: can't write 11 bytes to section .text of C:\Users\Dio\AppData\Local\Temp\ccY6j4zz.o: 'file too big'
C:/codeblocks/MinGW/bin/../lib/gcc/x86_64-w64-mingw32/13.1.0/../../../../x86_64-w64-mingw32/bin/as.exe: C:\Users\Dio\AppData\Local\Temp\ccY6j4zz.o: too many sections (41204)
C:\Users\Dio\AppData\Local\Temp\ccsHBHmj.s: Fatal error: C:\Users\Dio\AppData\Local\Temp\ccY6j4zz.o: file too big

That's the error i get when i run the command: "g++ cppfront.cpp -std=c++20 -o cppfront" on windows 7 using mingw g++ version 13.1.0

**Comment by DyXel** [comment]
Posts online say that you should be passing a flag for big objects to the assembler... Something along the lines of `g++ cppfront.cpp -std=c++20 -Wa,-mbig-obj -o cppfront`. Could you try that?

**Comment by hsutter** [comment]
Right, when I build with MSVC with some options I need to specify `/bigobj`.

Googling for "mingw /bigobj" led me to [this StackOverflow question](https://stackoverflow.com/questions/16596876/gcc-equivalent-of-mss-bigobj) where the chosen answer was: "The solution is to add the option `-Wa,-mbig-obj` if your version of GCC supports that option. You probably only need it during the compilation step, not the linker step. If your compiler does not support that option, you should look into using mingw-w64 and [MSYS2](http://msys2.github.io/)."
